components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -614,10 +584,6 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
     protected async findPrebuiltWorkspace(parentCtx: TraceContext, user: User, context: WorkspaceContext, mode: CreateWorkspaceMode): Promise<WorkspaceCreationResult | PrebuiltWorkspaceContext | undefined> {
         const ctx = TraceContext.childContextWithSpan("findPrebuiltWorkspace", parentCtx);
-        ctx.span?.addTags({
-            mode,
-            userId: user.id,
-        });
```

Comment:
Where's my answer gone??? Ok, here's the short version:
 - `this` inside `GitpodServerImpl` still points to the "regular" object
 - only API calls (public methods called from the websocket) have their parameters traced because they don't call `GitpodServerImpl` directly but through [this code](https://github.com/gitpod-io/gitpod/blob/952bc6dd24c4e682f4c3c3fd308cded64b93984d/components/server/src/websocket/websocket-connection-manager.ts#L285-L347)